subgraphs/spotify/src/resolvers/PlaybackItem.ts

```diff
@@ -0,0 +1,21 @@
+import { PlaybackItemResolvers } from "../__generated__/resolvers-types";
+
+export const PlaybackItem: PlaybackItemResolvers = {
+  __resolveType: (playbackItem) => {
+    switch (playbackItem.type) {
+      case "episode":
+        return "Episode";
+      case "track":
+        return "Track";
+    }
+  },
+  async __resolveReference(playbackItem, { dataSources }) {
+    //This is scritcly for a demo, not a good practice
```

Comment:
```suggestion
    // This is strictly for a demo, not a good practice
```

Is this comment related to the `try/catch` or the `id`? I'm guessing for the try/catch. Any chance that `playbackItem` has a `type` property on it that would allow us to avoid the potential double fetch?